rebasehelper/specfile.py

```diff
@@ -125,11 +127,35 @@ def __init__(self, path, sources_location='', download=True):
         rpm.addMacro("_sourcedir", self.sources_location)
         self._read_spec_content()
         # Load rpm information
-        self.spc = rpm.spec(self.path)
-        self.patches = self._get_initial_patches_list()
```

Comment:
I guess, you would like to explicitly call method _update_data, right?
SpecFile class will be only for initialization, right?

like
~~~
sp = SpecFile(something)
sp._update_data()
~~~

Did I understand it?